### PR TITLE
Update testHelpers.js

### PR DESCRIPTION
**Pull request by pieterbrandsen**
Small read update, I think myself it's better/cleaner in this way.

If you look at the other cases mostly it's done in this way instead of '+'.

### CODE REVIEW

File: utils/testHelpers.js [hunk 1]
```diff
@@ -204,7 +204,7 @@ const spawnBots = async function(line, socket, rooms, players, tickDuration) {
     await sleep(5);
 
     for (const room of rooms) {
-      console.log('> Spawn bot ' + room + ' as TooAngel');
+      console.log($`> Spawn bot ${room} as TooAngel`);
```
Comment (TooAngel) [inline utils/testHelpers.js:207]:
```suggestion
      console.log(`> Spawn bot ${room} as TooAngel`);
```

**Review by TooAngel — CHANGES_REQUESTED** [review]

**Review by TooAngel — APPROVED** [review]
Thx, for fixing it :-)

If you squash your commits to a single one it will get merged quite faster.

**Comment by pieterbrandsen** [comment]
No problem

**Comment by TooAngel** [comment]
This pull request was merged by [worlddriven](https://www.worlddriven.org).

**Pull request merged**